hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/PacketTest.java

```diff
@@ -65,7 +65,7 @@ public void setPacketType() {
             // COMPATIBILITY_BIND_MESSAGE occupies the same ordinal as SERVER_CONTROL
             // and COMPATIBILITY_EXTENDED_BIND occupies the same ordinal as SQL
             Type expected = type == Type.COMPATIBILITY_BIND_MESSAGE ? Type.SERVER_CONTROL
-                    : type == Type.COMPATIBILITY_EXTENDED_BIND ? Type.SQL
+                    : type == Type.COMPATIBILITY_EXTENDED_BIND ? Type.UNDEFINED5
                     : type;
```

Comment:
I would recommend future proofing this.

- We can use ordinal numbers for comments. `and COMPATIBILITY_EXTENDED_BIND occupies the same ordinal as SQL` -> `and COMPATIBILITY_EXTENDED_BIND occupies ordinal 5`
- We can use ordinal numbers for enums. `Type.UNDEFINED5` -> `Type.values()[5]`
- Same applies for `COMPATIBILITY_BIND_MESSAGE` and `SERVER_CONTROL`

Or we can just fix the comment from `SQL` to `UNDEFINED5`, but I believe in the future `UNDEFINED5` will change again.